app/connectors/httpParsers/NrsSubmissionHttpParser.scala

```diff
@@ -43,4 +43,4 @@ object NrsSubmissionHttpParser extends APIParser {
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
added empty line?

Reply:
Yes that was deliberate. I added a few like that to remove the Scala style warnings that specify the rule the end of file should have an empty line.